src/bin/cql-stress-cassandra-stress/settings/option/rate.rs

```diff
@@ -5,7 +5,7 @@ use crate::settings::{
 use anyhow::Result;
 
 pub struct RateOption {
-    threads_info: ThreadsInfo,
+    pub threads_info: ThreadsInfo,
```

Comment:
Is this change supposed to be included in this commit?

Reply:
Extracted to yet another commit